### PR TITLE
PR : Table Adjustment in BookingListComponent

### PR DESCRIPTION
**Pull request by Shivangi-Dixit**
- New Columns Added in BookingListComponent.
- Validation added for control Subject in AppointmentPartialComponent.

### CODE REVIEW

File: RealEstateCRM.PresentationLayer.WebApi/ClientApp/src/app/core/_config/i18n/en.ts [hunk 1]
```diff
@@ -65,6 +65,18 @@ export const locale = {
         INVALID_FIELD: 'Field is not valid',
       }
     },
+    BOOKING: {
+      list: {
+        label_id:'ID',
+        label_enquiryDate: 'Enquiry Date',
```
Comment (neersx) [inline RealEstateCRM.PresentationLayer.WebApi/ClientApp/src/app/core/_config/i18n/en.ts:71]: why this Enquiry date coming in booking?

File: RealEstateCRM.PresentationLayer.WebApi/ClientApp/src/app/views/pages/apps/booking/booking-list/booking-list.component.html [hunk 2]
```diff
@@ -39,34 +39,38 @@
 				matSortActive="bookingDate" matSortDirection="desc" matSortDisableClear>
 
 				<ng-container matColumnDef="id">
-					<mat-header-cell *matHeaderCellDef mat-sort-header>Id</mat-header-cell>
+					<mat-header-cell *matHeaderCellDef mat-sort-header>{{'BOOKING.list.label_id' | translate }}</mat-header-cell>
 					<mat-cell *matCellDef="let booking">{{booking.id}}</mat-cell>
 				</ng-container>
 				<ng-container matColumnDef="bookingDate">
-					<mat-header-cell *matHeaderCellDef mat-sort-header>Enquiry Date</mat-header-cell>
+					<mat-header-cell *matHeaderCellDef mat-sort-header>{{'BOOKING.list.label_enquiryDate' | translate }}</mat-header-cell>
```
Comment (neersx) [inline RealEstateCRM.PresentationLayer.WebApi/ClientApp/src/app/views/pages/apps/booking/booking-list/booking-list.component.html:46]: its not enquiry date.. its Booking Date

File: RealEstateCRM.PresentationLayer.WebApi/ClientApp/src/app/views/partials/content/forms/_appointment/appointment.partial.component.html [hunk 3]
```diff
@@ -16,6 +16,9 @@ <h3 class="card-label">{{getTitle()}}</h3>
                             <mat-hint align="start">Please enter
                                 <strong>Subject</strong>
                             </mat-hint>
+                            <mat-error>Subjet is
```
Comment (neersx) [inline RealEstateCRM.PresentationLayer.WebApi/ClientApp/src/app/views/partials/content/forms/_appointment/appointment.partial.component.html:19]: Subject spelling incorrect

**Review by neersx — CHANGES_REQUESTED** [review]
please review comments

File: RealEstateCRM.PresentationLayer.WebApi/ClientApp/src/app/core/_config/i18n/en.ts (hunk 1 above)
Comment (Shivangi-Dixit) [reply RealEstateCRM.PresentationLayer.WebApi/ClientApp/src/app/core/_config/i18n/en.ts]: Fixed

**Review by neersx — APPROVED** [review]

**Pull request merged**